meta-facebook/yv35-hd/src/platform/plat_isr.c

```diff
@@ -388,6 +404,15 @@ void ISR_APML_ALERT()
 		if ((status & 0x02) && (apml_write_byte(APML_BUS, SB_RMI_ADDR, SBRMI_STATUS, 0x02)))
 			LOG_ERR("Failed to clear SwAlertSts.");
 
-		send_apml_alert_to_bmc(ras_status);
+		fatal_error_work_info *delay_wrok = malloc(sizeof(fatal_error_work_info));
```

Comment:
s/wrok/work

Reply:
Hi Patrick,
Commit 8802b9d5a033833e91243fc479860a040764c090 correct the typo.
Please help to review again, thanks.